scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -212,21 +212,6 @@ Function AppendInstallationOption([string] $testCommand)
     Return $testCommand += " -a `"$IoTEdgedWorkingFolder`""
 }
 
-Function CleanUp
```

Comment:
this will break if someone just call Run-E2Etest.ps1 to run manually on Windows 10 Pro x64.  Please make sure the experience doesn't change as is for manual test run on Windows 10 Pro x64.  And please make it easier to manual run tests for IoT Core.

Reply:
Why would it break?
The functionality of function CleanUp got moved to Setup-Env.ps1 and the behavior for Windows 10 Pro x64 remains unchanged.

Reply:
Why would it break?
The functionality of ClenUp just got moved to Setup-Env.ps1 and remains the same.

Reply:
you currently call Setup-Env.ps1 in config yaml.  If I manually run e2e tests using Run-E2ETest.ps1, CleanUp will not be called. right?

Reply:
The _CleanUp_ code used to be called within _Function TestSetup_, which now calls _Setup-Env.ps1_ instead. The functionality of _Setup-Env.ps1_ for Windows Pro x64 remains the same as the former _CleanUp_.